ProteGO/UI/History/HistoryRootViewController/HistoryRootViewController.swift

```diff
@@ -0,0 +1,28 @@
+import UIKit
+
+protocol HistoryRootViewControllerDelegate: class {
+    func sendHistoryFinished(result: Result<Void, Error>)
+}
+
+final class HistoryRootViewController: UINavigationController {
```

Comment:
Moim zdaniem tu lepsze byłoby customowe rozwiązanie podmieniające kontrolery pod `BannerView`.

Reply:
A co nam to da, póki co ta nawigacja jest totalnie standardowa, wiec taki UINavigationController nam w zupełności wysarcza i daje dodatkowo ładne animacje przejść pomiędzy ekranami.

Reply:
Mielibyśmy spójność z onboardingiem i rejestracją (w kodzie i dla użytkownika) i reużywalne widoki bez odwoływania się do `self.navigationController`.
Co do animacji push, myślę że to mały plus, jeśli banner też jest pushowany. A w przypadku widoku ze spinnerem uważam, że nawet lepiej go pokazać bez pusha.

Reply:
ustaliliśmy poza GH, że zajmiemy się tym później: https://github.com/ProteGO-app/ios/issues/61